webpack.config.js

```diff
@@ -3,23 +3,24 @@
 /**
  * External dependencies
  */
-var webpack = require( 'webpack' ),
+let webpack = require( 'webpack' ),
```

Comment:
yeah, these are all from an auto "fixing" script I run on save. I'll see about fixing them up, though they're really just cluttering up the patch.